### PR TITLE
misc + iota axiom saves

### PR DESCRIPTION
**Pull request by icecream17**
Commit by commit (a lot more compartmentalized than my last pr)

About 40 to 50 miscallaneous downstream axiom saves are from the first few commits, and then iotaval/iotaex is responsible for the rest in the axiom diff.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -653832,6 +653837,25 @@ D Fn ( I ... ( M - 1 ) ) /\
     $( $j usage 'sn-dtru' avoids 'ax-8'; $)
   $}
 
+  ${
+    $d ph t $.  $d ps t $.  $d x t $.
+    $( Class abstractions in a subclass relationship, closed form.  One
+       direction of ~ ss2ab using fewer axioms.  (Contributed by SN,
```
Comment (jkingdon) [inline set.mm]: Can we add a `$j usage` to provide a bit more specificity to "fewer axioms"?

File: set.mm [hunk 2]
```diff
@@ -45178,21 +45178,8 @@ will do (e.g., ` O = (/) ` and ` T = { (/) } ` , see ~ 0nep0 ).
...
-       by SN, 1-Aug-2024.)  (Avoid depending on this detail.) $)
+       Carneiro, 26-Apr-2015.)  (Avoid depending on this detail.) $)
     dfopif $p |- <. A , B >. =
-      if ( ( A e. _V /\ B e. _V ) , { { A } , { A , B } } , (/) ) $=
-      ( vx vy cop cvv wcel wa csn cpr c0 cif cv w3a wn wo noel intnan 3bitr4i
-      wb biorf ax-mp df-3an orcom weq eleq1 3anbi3d df-op elab2g elv elif eqriv
-      ) CABEZAFGZBFGZHZAIABJJZKLZUNUOCMZUQGZNZUPUTHZUPOZUSKGZHZPZUSUMGZUSURGVBV
-      EVBPZVAVFVEOVBVHTVDVCUSQRVEVBUAUBUNUOUTUCVBVEUDSVGVATCUNUODMZUQGZNVADUSUM
-      FDCUEVJUTUNUOVIUSUQUFUGDABUHUIUJUPUSUQKUKSUL $.
-    $( $j usage 'dfopif' avoids 'ax-10' 'ax-11' 'ax-12'; $)
```
Comment (jkingdon) [inline set.mm:45190]: We want to keep the `$( $j usage 'dfopif' avoids 'ax-10' 'ax-11' 'ax-12'` even if we are restoring the old proof, right?

**Review by jkingdon — APPROVED** [review]
My suggestions are non-blocking.

The only thing in here which really caught my eye was the changes to iota related theorems. If someone who has thought more about iota than I has an opinion, speak up. If not, I think we can go with these changes, as they seem like improvements in terms of axiom usage.

File: set.mm [hunk 3]
```diff
@@ -653815,19 +653911,40 @@ D Fn ( I ... ( M - 1 ) ) /\
...
       CGZCDEZFZDGCGUQVCUSCGVADGACKDALUSVACDMNVBVECDVDVBVDUSVAFZQVBFVDUSUTAGVFVD
       URUTACDAOPUTAUAUBUSVAUCUDUEUFVEUQCDDBEZVEUQQVGVECBEZFZUQVGVHVDDCBUGRVIUPA
       CACEUOVHACBSRTUIVGFZUQVEVJUPADADEUOVGADBSRTUHUJUKULUOAUMUN $.
+    $( $j usage 'sn-dtru' avoids 'ax-8'; $)
+  $}
+
+  ${
+    $d ph t $.  $d ps t $.  $d x t $.
+    $( Class abstractions in a subclass relationship, closed form.  One
+       direction of ~ ss2ab using fewer axioms.  (Contributed by SN,
```
Comment (jkingdon) [inline set.mm:653934]: Can we add a $j usage to provide a bit more specificity to "fewer axioms"?

File: set.mm (hunk 1 above)
Comment (jkingdon) [reply set.mm]: Something about the github mechanics led to this comment being called "outdated" so I made another comment. But both of them are about `ss2ab1` so you only need to pay attention to them once.

**Comment by icecream17** [comment]
~~blocked until #4503 is merged~~

File: set.mm [hunk 4]
```diff
@@ -653815,19 +653815,21 @@ D Fn ( I ... ( M - 1 ) ) /\
...
     $d w x y z $.
     $( A version of ~ el with an inner existential quantifier on ` x ` , which
-       avoids ~ ax-7 and ~ ax-8 .  (Contributed by SN, 18-Sep-2023.) $)
+       only uses ~ ax-mp to ~ ax-4 , ~ ax-6 , and ~ ax-pr .  (Contributed by
+       SN, 18-Sep-2023.) $)
     sn-el $p |- E. y E. x x e. y $=
-      ( vw vz wel wi wal wex ax-pow weq ax6ev ax9v1 alrimiv eximii exim mpi ) C
-      AECDEFZCGZABEZFAGZSAHZBDBACITRAHUAADJZRAADKUBQCADCLMNRSAOPN $.
+      ( vz weq wo wel wi wal wex ax-pr ax6ev pm2.07 eximii exim mpi ) ACDZPEZAB
+      FZGAHZRAIZBCCBAJSQAITPQAACKPLMQRANOM $.
```
Comment (benjub) [inline set.mm]: Can you keep both versions ?  The old one is not completely subsumed and may be of interest.

**Review by benjub — APPROVED** [review]
Thanks @icecream17 for the careful updates !

**Pull request merged**